### PR TITLE
Labels info

### PR DESCRIPTION
**Pull request by t-rutten**
Add `labels_info/0` to CIFAR100, CIFAR10, and FashionMNIST to show descriptions of dataset labels.

### CODE REVIEW

File: lib/scidata/cifar100.ex [hunk 1]
```diff
@@ -61,6 +61,100 @@ defmodule Scidata.CIFAR100 do
...
+          [15, 29],
+          [4, 0],
+          [14, 11],
+          [1, 1],
+          ...
+        ]
+      >
+      iex> coarse_labels = labels_tensor |> Nx.slice([0,0], [50000, 1]) \
+      |> Nx.reshape({50000}) |> Nx.to_flat_list() \
+      |> Enum.map(fn label_index -> Enum.at(coarse, label_index) end)
```
Comment (t-rutten) [inline lib/scidata/cifar100.ex]: Is there a more concise, more idiomatic equivalent transformation in this example?

File: lib/scidata/cifar10.ex [hunk 2]
```diff
@@ -52,6 +52,48 @@ defmodule Scidata.CIFAR10 do
...
+    labels =
+      files
+      |> Enum.find(fn {fname, _} ->
+        String.match?(List.to_string(fname), ~r/batches.meta/)
+      end)
+      |> elem(1)
+      |> String.trim_trailing()
+      |> String.split("\n")
+
+    labels
```
Comment (seanmor5) [inline lib/scidata/cifar10.ex:92]: Do you think maybe this could return a map so you don't have to index into a potentially large list every time?

**Review by josevalim — APPROVED** [review]

File: lib/scidata/cifar10.ex [hunk 3]
```diff
@@ -52,6 +52,48 @@ defmodule Scidata.CIFAR10 do
     download_dataset(:train, opts)
   end
 
+  @doc """
+  Shows descriptions of dataset labels.
+
+  Label values returned by `download/1` correspond to indices in the lists
+  returned here.
+
+  ## Examples
```
Comment (josevalim) [inline lib/scidata/cifar10.ex:61]: Please add a new line after section headers. :)

File: lib/scidata/cifar10.ex [hunk 4]
```diff
@@ -52,6 +52,48 @@ defmodule Scidata.CIFAR10 do
...
+      iex> label_names = Scidata.CIFAR10.labels_info()
+      ["airplane", "automobile", "bird", "cat", "deer", "dog", "frog", "horse",
+       "ship", "truck"]
+      iex> labels |> Nx.to_flat_list() |> Enum.map(fn label_index -> Enum.at(label_names, label_index) end)
+      ["frog", "truck", "truck", "deer", "automobile", "automobile", "bird", "horse",
+       "ship", "cat", "deer", "horse", "horse", "bird", "truck", "truck", "truck", ...]
+
+
+
+
```
Comment (josevalim) [inline lib/scidata/cifar10.ex]:
```suggestion
```

File: lib/scidata/cifar100.ex (hunk 1 above)
Comment (josevalim) [reply lib/scidata/cifar100.ex]: This example is definitely doing too much... so I think we should rather remove the example for now or find a cleaner way. :)

File: lib/scidata/fashionmnist.ex [hunk 5]
```diff
@@ -69,6 +71,28 @@ defmodule Scidata.FashionMNIST do
      download_labels(@test_label_file, transform_labels)}
   end
 
+  @doc """
+  Shows descriptions of dataset labels.
+
+  ## Examples
```
Comment (josevalim) [inline lib/scidata/fashionmnist.ex:77]:
```suggestion
  ## Examples

```

File: lib/scidata/cifar10.ex (hunk 2 above)
Comment (t-rutten) [reply lib/scidata/cifar10.ex:92]: We could do that--a map like `%{0 => "airplane", 1 => "automobile", ...}`?

**Comment by josevalim** [comment]
Oh, something I want to add is that we eventually want to add this kind of pretty printing to Nx directly as part of FeatureSet. So perhaps it makes sense to hold on merging this until that work is done? 🤔

**Comment by t-rutten** [comment]
@josevalim sure thing, let's wait on this until we can integrate with FeatureSet.

**Pull request closed without merging**